src/modlunky2/assets/patcher.py

```diff
@@ -75,15 +75,14 @@ def patch_checksum(self):
                 "to be updated for the current game version."
             )
             logger.warning(
-                "(Expected 0x{:02x}, found 0x{:02x})".format(
-                    CHECKSUM_PATCH_END, ops[-1]
-                )
+                f"(Expected 0x{CHECKSUM_PATCH_END:02x}, found 0x{ops[-1]:02x})"
```

Comment:
Did the linter really request using fstrings for logging? That's historically considered an anti pattern because you pay for the interpolation even if you're not logging at that log level.

Reply:
Then again, format was already doing that here.

Reply:
Yeah, and pylint complains about not using % for logging both before and after this change.

I didn't look into whether this "eager interpolation" is working around an issue, or is just poor style.

Reply:
Likely just a mistake initially